src/rust/engine/process_execution/Cargo.toml

```diff
@@ -43,18 +43,20 @@ lazy_static = "1"
 parking_lot = "0.12"
 itertools = "0.10"
 serde = "1.0.136"
-bincode = "1.3.3"
 async-oncecell = "0.2"
+bincode = "1.3.3"
 once_cell = "1.15"
-rand = "0.8"
+os_pipe = "1.0"
 prost = "0.9"
 prost-types = "0.9"
+rand = "0.8"
 strum = "0.24"
 strum_macros = "0.24"
 tonic = { version = "0.6", features = ["transport", "codegen", "tls", "tls-roots", "prost"] }
 tryfuture = { path = "../tryfuture" }
 
```

Comment:
Given how things have been reorganized, you should be able to prune this list of dependencies: `async-oncecell`, `bollard`, `nails`, `tokio-util`, `tonic`, and hopefully a few others.